subcommittees/mentoring/procedure-debriefing-session.md

```diff
@@ -0,0 +1,41 @@
+# Procedure - Debriefing Session
+
+This document intends to describe how debriefing sessions
+are organized so that it continue even after members go away.
+
+## Terminology
+
+-   **Debriefing session**: is an online meeting
+    where instructors could share experience about the workshops
+    they taught.
+
+## We Need a Debriefing Session
+
+The Mentoring Subcommittee is responsible to organize
+the debriefing sessions so that
+
+-   each month has at least one debriefing session and
+
+-   instructors are notify with at least one week in advance.
+
```

Comment:
...possibly add purpose of meeting (aka why we have them) and not only practicalities?